clang/docs/ReleaseNotes.rst

```diff
@@ -402,6 +402,10 @@ Improvements to Clang's diagnostics
 - Clang now emits a diagnostic in case `vector_size` or `ext_vector_type`
   attributes are used with a negative size (#GH165463).
 
+- Clang now emits an extension warning (`ExtWarn`) instead of a regular warning
+  for static local variables declared inside `extern inline` functions
+  (#GH39524).
```

Comment:
```suggestion
- The warning about static local variables declared inside `extern inline` 
  functions is now correctly converted to an error if `-pedantic-errors` is 
  passed (#GH39524).
```
`ExtWarn` etc. is a Clang implementation detail; that shouldn’t be in a release note.